### PR TITLE
More cleanup

### PR DESCRIPTION
**Pull request by nbrady-techempower**
Removed more .gitignore files
Removed `toolset/setup/linux`
Updated the `README` and included changes missed from https://github.com/TechEmpower/FrameworkBenchmarks/commit/e9185159601df023e1472c021f05785c50da9d23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,31 +9,24 @@ If you're new to the project, welcome! Please feel free to ask questions [here](
...
 [test requirements section](https://frameworkbenchmarks.readthedocs.org/en/latest/Project-Information/Framework-Tests/).
 
 If you find yourself in a directory or file that you're not sure what the purpose is, checkout our [file structure](http://frameworkbenchmarks.readthedocs.org/en/latest/Codebase/#file-structure) in our documenation, which will briefly explain the use of relevant directories and files.
 
 ## Quick Start Guide
 
-Get started developing quickly by utilizing vagrant with TFB. [Git](https://git-scm.com), 
-[Virtualbox](https://www.virtualbox.org/) and [vagrant](https://www.vagrantup.com/) are 
-required.
+To get started developing you'll need to install [docker](https://docs.docker.com/install/).
```
Comment (msmith-techempower) [inline README.md]: Maybe something here about "or optionally install VirtualBox and Vagrant" and link to that header in this README.md

File: README.md [hunk 2]
```diff
@@ -9,31 +9,24 @@ If you're new to the project, welcome! Please feel free to ask questions [here](
...
 
 1. Clone TFB.
 
         $ git clone https://github.com/TechEmpower/FrameworkBenchmarks.git
 
-2. Change directories
-
-        $ cd FrameworkBenchmarks/deployment/vagrant
-
-3. Create the TFB Docker virtual network
```
Comment (msmith-techempower) [inline README.md]: I did not even occur to me that we specify the source directory as a part of the run, so being in this directory is not required. 👍

File: frameworks/C++/poco/.gitignore [hunk 3]
```diff
@@ -1 +0,0 @@
-/poco
```
Comment (msmith-techempower) [inline frameworks/C++/poco/.gitignore]: Just remove this file?

Comment (nbrady-techempower) [reply frameworks/C++/poco/.gitignore]: this is what it looks like when a file is removed

**Comment by nbrady-techempower** [comment]
the two php errors are unrelated, and ill look in to those separately

**Pull request merged**